compiler/pipes/generate-virtual-methods.cpp

```diff
@@ -11,6 +11,7 @@
 #include "compiler/data/function-data.h"
 #include "compiler/data/src-file.h"
 #include "compiler/gentree.h"
+#include "compiler/vertex-util.h"
```

Comment:
gentree.h still needed?

Reply:
Yes. It uses `GenTree::create_switch_vertex` which is only needed inside gentree and 1 more place: this file.
I tried to move only things that are generally useful.
`create_switch_vertex` could be a candidate too, but I avoided it for the first iteration.